src/App.js

```diff
@@ -39,8 +40,25 @@ class App extends Component {
     let { currentDataset } = this.state;
     return (
       <div className="App">
-        { this.renderVocabButtons() }
+        <div id="language-sets-container">
+          { this.renderVocabButtons() }
+        </div>
+        <div id="header">VOCABULARY STUDY BUDDY</div>
+        <div id="instructions">
+          <p>
+            Goal: Match each English word with its translation 
+            in the currently selected language (e.g. "hello" with "hola") 
+            by clicking the appropriate cards. 
+            Incorrect selections will flash a red border 
+            while correct ones will flash a green border. 
+            You may reset and/or reshuffle the board at any time 
+            by clicking the red reset button, 
+            and you may switch between languages using 
+            the buttons at the top of the page.
+          </p>
+        </div>
```

Comment:
good start for instructional text. some comments:
- text is kinda small
- its a block of text, so hard to really quickly get a sense of key instructions (remember most ppl will just skim/glance), so instructions should be bigger, and obvious what should be done

Reply:
Do you have suggestions or edits for changes then? This is the most pared down I could think of that still explains all the parts to the game.

Reply:
add a <br /> within the text. its just one block of text -- i think it needs to have line breaks between some of the sentences so its easier to skim